components/helper/ConfirmModal.tsx

```diff
@@ -0,0 +1,35 @@
+import { Box, Button, Modal, Paper, Typography }
+  from '@mui/material';
+
+function ConfirmModal({
+  open,
+  onClose,
+  message,
+  onConfirm,
+  onCancel,
+  confirmButtonName,
+  }) {
+
+  return (
+    <Modal open={open} onClose={onClose} id='modalConfirm'>
```

Comment:
The `id` should be Kebab Case (use a hyphen `-` for separating the names). Same for any other `id`.